packages/eslint-config-react-app/index.js

```diff
@@ -79,6 +79,16 @@ module.exports = {
       'no-array-constructor': 'off',
       '@typescript-eslint/no-array-constructor': 'warn',
       '@typescript-eslint/no-namespace': 'error',
+      '@typescript-eslint/no-use-before-define': [
+        'warn',
+        {
+          functions: false,
+          classes: false,
+          variables: false,
+          typedefs: false,
+        },
+      ],
+      'no-use-before-define': 'off',
```

Comment:
Do you mind flipping the order of `@typescript/no-use-before-define` and `no-use-before-define` for consistency with the other rules here?

Reply:
Not at all. Done.